code/modules/antagonists/_common/antag_spawner.dm

```diff
@@ -134,35 +137,26 @@
 	var/mob/living/carbon/human/M = new/mob/living/carbon/human(T)
 	C.prefs.copy_to(M)
 	M.key = C.key
+	var/datum/mind/op_mind = M.mind
 
-	var/datum/antagonist/nukeop/new_op = new()
-	new_op.send_to_spawnpoint = FALSE
-	new_op.nukeop_outfit = /datum/outfit/syndicate/no_crystals
+	antag_datum = new()
+	antag_datum.send_to_spawnpoint = FALSE
+	antag_datum.nukeop_outfit = outfit
 
-	var/datum/antagonist/nukeop/creator_op = user.has_antag_datum(/datum/antagonist/nukeop,TRUE)
+	var/datum/antagonist/nukeop/creator_op = user.has_antag_datum(antag_datum, TRUE)
```

Comment:
you reverted your own fix with this since you're checking for the clown nukeop datum specifically again.

```
	var/datum/antagonist/nukeop/creator_op = user.has_antag_datum(/datum/antagonist/nukeop, TRUE)
	op_mind.add_antag_datum(new_op, creator_op ? creator_op.get_team() : null)
	op_mind.special_role = special_role_name
```

Reply:
That is hilarious